plugins/synaptics-vmm9/README.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Plugin: Synaptics VMM9xxxx
+---
```

Comment:
Why does this need it's own plugin?  Can't go into regular synaptics mst plugin?

Reply:
I actually started doing that (`hughsie/FuSynapticsMstHidDevice` branch), but there was so much special casing it was making something already complicated kinda insane. The only bit common now is the error and command enums in the .rs file, and if you agree I can remove the *no longer supported* commands and errors.

Reply:
I think it's fine to have them separate now that I see how it works.  How about synaptics-mst-usb or something like that to indicate this one works over HID over USB?

Reply:
I think synaptics-mst works for now, i.e. vendor-protocol -- but the new plugins seems to have a `vendor-chipset` which I like better. I think if we did the synaptics-mst again I'd have plugins for leaf/tesla, panamera and cayenne/spyder.